src_new/adapters/prebid.js

```diff
@@ -227,6 +227,15 @@ function isPubMaticAdUnitPresentForDivId(adUnits, divID){
 exports.isPubMaticAdUnitPresentForDivId = isPubMaticAdUnitPresentForDivId;
 /* end-test-block */
 
+/* start-test-block */
+exports.getMediaTypeObject = getMediaTypeObject;
```

Comment:
please move export statements below function declaration, just for consistency in code

Reply:
As discussed removed this function